pkgs/development/python-modules/requests/0001-Prefer-NixOS-Nix-default-CA-bundles-over-certifi.patch

```diff
@@ -0,0 +1,41 @@
+From 34087798ef2167636c93a939fb9488fa5533ccd8 Mon Sep 17 00:00:00 2001
+From: Keshav Kini <keshav.kini@gmail.com>
+Date: Sun, 16 May 2021 20:35:24 -0700
+Subject: [PATCH] Prefer NixOS/Nix default CA bundles over certifi
+
+---
+ requests/certs.py | 18 +++++++++++++++++-
+ 1 file changed, 17 insertions(+), 1 deletion(-)
+
+diff --git a/requests/certs.py b/requests/certs.py
+index d1a378d7..36e78b5d 100644
+--- a/requests/certs.py
++++ b/requests/certs.py
+@@ -12,7 +12,23 @@ If you are packaging Requests, e.g., for a Linux distribution or a managed
+ environment, you can change the definition of where() to return a separately
+ packaged CA bundle.
+ """
+-from certifi import where
++
++import os
++
++import certifi
++
++
++def where():
++    nix_ssl_cert_file = os.getenv("NIX_SSL_CERT_FILE")
++    if nix_ssl_cert_file and os.path.exists(nix_ssl_cert_file):
++        return nix_ssl_cert_file
++
++    NIXOS_CA_BUNDLE = "/etc/ssl/certs/ca-certificates.crt"
++    if os.path.exists(NIXOS_CA_BUNDLE):
++        return NIXOS_CA_BUNDLE
```

Comment:
```suggestion
+    nixos_ca_bundle = "/etc/ssl/certs/ca-certificates.crt"
+    if os.path.exists(nixos_ca_bundle):
+        return nixos_ca_bundle
```

Reply:
PEP 8 says that constants are "usually" defined at the module level, but I figured these count as constants too, though they're local to the function. Still, I'll follow your suggestion since I'm not religious about it or anything :)

Reply:
Constants would be declared at the top, usually after imports.

This is just inside the `where()` function, and used as variables. So I think @mweinelt 's suggestion is appropriate.